src/base/system_parameter.c

```diff
@@ -729,13 +729,8 @@ static const char sysprm_ha_conf_file_name[] = "cubrid_ha.conf";
 
 #define PRM_NAME_STATDUMP_FORCE_ADD_INT_MAX "statdump_force_add_int_max"
 
-#ifndef NDEBUG
-#define PRM_NAME_USE_DEDUPLICATE_KEY_MODE_OID_TEST  "use_deduplicate_key_mode_oid_test"
-#endif
-#define PRM_NAME_DEDUPLICATE_MIN_KEYS      "deduplicate_min_keys"
-#define PRM_NAME_DEDUPLICATE_FK_LEVEL      "deduplicate_fk_level"
 #define PRM_NAME_DEDUPLICATE_KEY_LEVEL     "deduplicate_key_level"
-#define PRM_NAME_DEDUPLICATE_PRINT_LEVEL   "deduplicate_print_level"
+#define PRM_NAME_WITH_OPTION_PRINT         "with_option_print"
```

Comment:
```suggestion
#define PRM_NAME_PRINT_WITH_CLAUSE         "print_with_clause"
```

Reply:
We will change each as follows.
print_index_detail, --skip-index-detail

thanks a lot.